lib/reporters/error-reporter.js

```diff
@@ -0,0 +1,37 @@
+const Logger = require('../logger');
+const colors = require('colors');
+const path = require('path');
+const isArray = require('lodash/isArray');
+
+const logger = new Logger();
```

Comment:
as mentioned in https://github.com/voorhoede/fastatic/pull/97/files#r78731535 I think you may be able to do this without the `logger`, just regular `console.log`.